src/main/webapp/resources/js/pages/launch/LaunchFiles.jsx

```diff
@@ -0,0 +1,186 @@
+          }
+          firstSamples.push(sample);
+        });
+
+        setSamples(firstSamples);
+        setSelected(firstSelected);
+      })
+      .catch((message) => notification.error({ message }));
+  }, [paired, singles]);
+
+  /*
+  Called when a user selects a different set of files to run on the sample.
+  Needs to be updated through the reducer so that the correct file ids are
+  set when the pipeline is launched.
+   */
+  React.useEffect(() => {
+    setSelectedSampleFiles(dispatch, selected);
+  }, [dispatch, selected]);
+
+  /*
+  Calculate the div size if the number of samples changes
+   */
+  React.useEffect(() => {
+    if (samples) {
+      let newHeight = 0;
+      for (let i = 0; i < samples.length; i++) {
+        newHeight += getRowHeight(i);
+        if (newHeight > 600) {
+          newHeight = 600;
+          break;
+        }
+      }
+      setHeight(newHeight + 2);
+    }
+  }, [samples]);
+
+  /*
+  Called independently for each sample when the selected file set is changed.
+   */
+  const updateSelectedFiles = (sample, current) => {
+    const ids = new Set(selected);
+    ids.delete(sample.selected);
+    sample.selected = current;
+    ids.add(current);
+    setSelected(Array.from(ids));
+  };
+
+  /*
+  Called to remove a sample from the cart.
+   */
+  const removeSampleFromCart = (sample, selectedId) => {
+    removeSample(sample.project.id, sample.id).then(() => {
+      // Find the index so that we can update the table heights
+      const index = samples.findIndex((s) => s.id === sample.id);
+
+      const updatedSamples = samples.filter((s) => s.id !== sample.id);
+      if (updatedSamples.length) {
+        setSamples(updatedSamples);
+        const ids = new Set(selected);
+        ids.delete(selectedId);
+        setSelected(Array.from(ids));
+
+        // Update the virtual list
+        listRef.current.resetAfterIndex(index, true);
+      }
+    });
+  };
+
+  /**
+   * Create the react component to render a sample with it's files.
+   * @param index - index of the sample
+   * @param style - passed from VirtualListComponent
+   * @returns {JSX.Element}
+   */
+  const generateSample = ({ index, style }) => {
+    const sample = samples[index];
+    return (
+      <SampleFilesListItem
+        style={style}
+        sample={sample}
+        removeSample={removeSampleFromCart}
+        updateSelectedFiles={updateSelectedFiles}
+      />
+    );
+  };
+
+  return (
+    <Space
+      className="t-launch-files"
+      direction="vertical"
+      style={{ width: `100%` }}
+    >
+      <SectionHeading id="launch-files">
+        {i18n("LaunchFiles.heading")}
+      </SectionHeading>
+      {samples ? (
+        <div
+          style={{
+            height,
+            width: "100%",
+          }}
+        >
+          <AutoSizer>
+            {({ height = 600, width = 400 }) => (
+              <VList
+                style={{
+                  border: `1px solid ${grey4}`,
```

Comment:
Can you update this to use `BORDERED_LIGHT` from the `borders.js file` as we already have this exact style setting set

Reply:
Thanks, I actually forgot about that.  Updated in 8e049f9aaaa8b9428af69f1b316dc07bd7e9eb69